integration_tests/test_aggregator_ess.py

```diff
@@ -0,0 +1,125 @@
+import logging
+
+import json
+import traceback
+
+from d3a_api_client.redis_aggregator import RedisAggregator
+from d3a_api_client.redis_device import RedisDeviceClient
+
+
+class EssAggregator(RedisAggregator):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.errors = 0
+        self.status = "running"
+        self._setup()
+        self._has_tested_bids = False
+        self._has_tested_offers = False
```

Comment:
A reusable class for these tests defining all the repetitive properties would be great

Reply:
Great idea! Will do that.